src/topology/compact_open.lean

```diff
@@ -58,22 +58,22 @@ topological_space.generate_open.basic _ (by dsimp [mem_set_of_eq]; tauto)
 
 section functorial
 
-variables {g : β → γ} (hg : continuous g)
+variables (g : C(β, γ))
 
-def induced (f : C(α, β)) : C(α, γ) := ⟨g ∘ f, hg.comp f.continuous⟩
+def induced (f : C(α, β)) : C(α, γ) := g.comp f
```

Comment:
```suggestion
```
Shouldn't we just drop `continuous_map.induced` entirely since it's just another name for `continuous_map.comp`?